modules/core/settings/actions.js

```diff
@@ -88,3 +96,74 @@ export function getLanguage(event) {
         }
     }
 }
+
+export function toggleSwarm(event) {
+    return (dispatch, getState) => {
+        if ([SwarmState.Enabled, SwarmState.Enabling].indexOf(getState().settings.swarmState) > -1) {
```

Comment:
ES7(?) added the `includes` method on the Array prototype. so,
`[x, y].includes(getState().settings.swarmState)`
cleans that up a little.